### DISCUSSION TITLE
xorgxrp PGP key

### DISCUSSION THREAD

**Discussion by Yrosenz** [neutrinolabs/xrdp#2148] [General]
Metalefty, I'm trying to build xorgxrdp package for arch linux but makepkg fails your PGP key.
Here is the output:

makepkg
==> Making package: xorgxrdp 0.2.18-1 (Mon 14 Feb 2022 08:30:39)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found xorgxrdp-0.2.18.tar.gz
  -> Found xorgxrdp-0.2.18.tar.gz.asc
==> Validating source files with sha256sums...
    xorgxrdp-0.2.18.tar.gz ... Passed
    xorgxrdp-0.2.18.tar.gz.asc ... Skipped
==> Verifying source file signatures with gpg...
    xorgxrdp-0.2.18.tar.gz ... FAILED (unknown public key 03993B4065E7193B)
==> ERROR: One or more PGP signatures could not be verified!
Don't know if it helps but here is the PKGBUILD file.
[PKGBUILD.txt](https://github.com/neutrinolabs/xrdp/files/8058423/PKGBUILD.txt)

**Comment by metalefty** [comment]
See https://github.com/neutrinolabs/xorgxrdp/issues/207.
